topic-operator/src/main/java/io/strimzi/operator/topic/v2/BatchingTopicController.java

```diff
@@ -112,6 +113,13 @@ public class BatchingTopicController {
                 || !"false".equals(kt.getMetadata().getAnnotations().get(MANAGED));
     }
 
+    /* test */ static boolean isPaused(KafkaTopic kt) {
+        return kt.getMetadata() != null
+            && kt.getMetadata().getAnnotations() != null
+            && kt.getMetadata().getAnnotations().get(PAUSED) != null
+            && "true".equals(kt.getMetadata().getAnnotations().get(PAUSED));
```

Comment:
```suggestion
        return Annotations.isReconciliationPausedWithAnnotation(kt);
```

Or maybe even better ... remove the method and use the call directly instead.